atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DbKeyValueServiceConfig.java

```diff
@@ -50,6 +50,12 @@ public final String type() {
         return DbAtlasDbFactory.TYPE;
     }
 
+    @Override
+    @Value.Default
+    public int maxConcurrentGetRanges() {
+        return Math.max(2 * connection().getMaxConnections() / 3, 1);
```

Comment:
Is there any reason why its 2/3rd for DbKvs but 100% for Cassandra?

Reply:
Because Cassandra can burst much higher than the `poolSize` so it won't actually use all possible connections. The 2/3 was arbitrary, though, so open to suggestions.

Reply:
That should depend on the transaction patterns of the service, since this is configurable should be fine to leave it as 2/3rd due to lack of a better default value.